addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -13,16 +13,38 @@
 
     @include print-variable('priority-palette-prefix', $o-palette-priority-prefix);
 
-    // Need the colors and name of each color palette
+    // Need the colors, name, menu background color and footer background color of each color palette
     $palette-names: ();
+    $default-presets: ();
+    $bg-attrs: 'menu', 'footer';
+    @each $attr in $bg-attrs {
+        $default-presets: map-merge($default-presets, ($attr: map-get($o-base-color-palette, $attr)))
+    }
+
     @each $palette-name, $palette in $o-color-palettes {
         @if $palette-name != 'user-palette' {
             @each $key, $value in $palette {
                 @include print-variable('o-palette-#{$palette-name}-#{$key}', $value);
             }
             $palette-names: append($palette-names, $palette-name);
+
+            @each $attr-name, $default-preset-idx in $default-presets {
+                $preset-idx: o-safe-get($palette, $attr-name, $default-preset-idx);
+                $preset: nth($o-color-combinations, $preset-idx);
```

Comment:
`$preset-idx` can be equal to `null` here, probably making this line crash. More annoying: null here means that the logo has no default color (transparent) but that could be overridden by the `menu-custom` / `footer-custom` variables :grimacing: And in fact, even if the value is not null, the bg color is still overridden by menu-custom or footer-custom if mentioned... it could even be overridden by menu-gradient or footer-gradient :grimacing: 

... I guess we could compromise and ignore xxx-custom and xxx-gradient but in that case, there should be a comment to explain it + the 'null' value for the xxx key should default to the preset 1 (instead of crashing).

Reply:
I don't see how `$preset-idx` could be null. It means that `$default-preset-idx` must be `null`. But `$default-preset-idx` for menu and footer come from the `$o-base-color-palette` where `menu` and `footer` are not `null`. What do
![Screenshot from 2021-09-06 13-59-37](https://user-images.githubusercontent.com/49784121/132214461-c3ded7c2-663e-478f-a86e-9a63cdae5e08.png)
 I miss?

Reply:
> It means that $default-preset-idx must be null

Using `o-safe-get` there is the first error indeed: you ignore the null value coming from the palette but the palette should actually be able to define "null" and not fallbacking to the default value -> but this is indeed not working right now, but it should. I'll fix that. The point is that the final value of a color palette can be null.

> But $default-preset-idx for menu and footer come from the $o-base-color-palette where menu and footer are not null

This could also change in the future, your code here should not rely on you knowing the current value is currently not null.